docs/UserGuide.md

```diff
@@ -70,6 +70,22 @@ Examples:
 * `add n/Fifty Shades of Gray total/350 bookmark/200`
 * `add n/Harry Potter and the Chamber of Secrets total/1500 bookmark/25`
 
+### Viewing details of a book : `View`
+
+Shows a list of all books in the Storage Library.
```

Comment:
I think you may have missed out on this part. The View command is meant to view detailed info regarding a single book.

Reply:
Thanks for spotting it! 

Updated the PR, please help me review again. Thanks!